### PR TITLE
feat: Add custom webhook as notification option

### PR DESCRIPTION
**Pull request by coolusaHD**
Add the support of custom webhooks as notification endpoint

This would be the first step to solve [upptime discussions 191](https://github.com/orgs/upptime/discussions/191)

It uses `NOTIFICATION_WEBHOOK` boolean and
`NOTIFICATION_WEBHOOK_URL` for the URL

Other configuration could be possible when this PR gets merged.

Thanks ✌️

### CODE REVIEW

**Comment by coolusaHD** [comment]
👀  FYI 
@AnandChowdhary

File: src/helpers/notifme.ts [hunk 1]
```diff
@@ -391,4 +391,23 @@ export const sendNotification = async (message: string) => {
     }
     console.log("Finished sending Microsoft Teams");
   }
+  if (getSecret("NOTIFICATION_WEBHOOK")) {
```
Comment (AnandChowdhary) [inline src/helpers/notifme.ts:394]:
```suggestion
  if (getSecret("NOTIFICATION_CUSTOM_WEBHOOK")) {
```

File: src/helpers/notifme.ts [hunk 2]
```diff
@@ -391,4 +391,23 @@ export const sendNotification = async (message: string) => {
...
+    console.log("Sending Webhook");
+    try {
+      await axios.post(`${getSecret("NOTIFICATION_WEBHOOK_URL")}`,{
```
Comment (AnandChowdhary) [inline src/helpers/notifme.ts:397]:
```suggestion
      await axios.post(`${getSecret("NOTIFICATION_CUSTOM_WEBHOOK_URL")}`,{
```

**Review by AnandChowdhary — CHANGES_REQUESTED** [review]
Perhaps it makes sense to rename `NOTIFICATION_WEBHOOK` to `NOTIFICATION_CUSTOM_WEBHOOK`?

For example, for Microsoft Teams we have:
- `NOTIFICATION_TEAMS`
- `NOTIFICATION_TEAMS_WEBHOOK_URL`

So maybe we can make it clear that it's a custom webhook rather than a service webhook?

**Comment by AnandChowdhary** [comment]
Once merged, can you also please add docs to https://github.com/upptime/upptime.js.org/blob/master/docs/notifications.md?

**Comment by coolusaHD** [comment]
@AnandChowdhary 
See https://github.com/upptime/upptime.js.org/pull/55

And thanks for the quick response

**Pull request merged**